mods/ra2/rules/allied-naval.yaml

```diff
@@ -212,3 +212,53 @@ dlph:
 	HitShape:
 	OwnerLostAction:
 		Action: Kill
+
+carrier:
+	Inherits: ^Ship
+	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
+	Inherits@MC: ^MindControllable
+	Buildable:
+		Queue: Ship
+		BuildPaletteOrder: 60
+		Prerequisites: gatech, ~gayard
+		Description: Aircraft carrier ship.\n  Strong vs Tanks, Structures\n  Weak vs Infantry
+	Valued:
+		Cost: 2000
+	Tooltip:
+		Name: Aircraft Carrier
+	Health:
+		HP: 800
+	Armor:
+		Type: Heavy
+	RevealsShroud:
+		Range: 7c0
+		Type: CenterPosition
+	Selectable:
+		Bounds: 60,80,0,-6
+	Mobile:
+		TurnSpeed: 1
+		Speed: 60
+	RevealsShroud:
+		Range: 7c0
+	Turreted:
+		TurnSpeed: 255
+	AttackTurreted:
```

Comment:
Why is this turreted? Do we really have no better way of letting things attack regardless of their facing?

Reply:
In RV Aircraft Carrier is AttackFrontal with 128 FacingTolerance.